scipy/signal/windows/windows.py

```diff
@@ -1620,6 +1620,80 @@ def exponential(M, center=None, tau=1., sym=True):
     return _truncate(w, needs_trunc)
 
 
+def taylor(M, nbar=4, level=-30, norm=True, sym=True):
+    """
+    Return the Taylor window.
+
+    The Taylor window allows for a selectable sidelobe suppression with a
+    minimum broadening. This window is commonly used in radar processing [1]_.
+
+    Parameters
+    ----------
+    M : int
+        Number of points in the output window. If zero or less, an
+        empty array is returned.
+    nbar : int
+        Number of nearly constant level sidelobes adjacent to the mainlobe.
+    level : float
+        Desired peak sidelobe level in decibels (db) relative to the mainlobe.
+    norm : boolean
+        When True (default), normalizes the window such that all values are
+        less than or equal to 1.
+    sym : bool, optional
+        When True (default), generates a symmetric window, for use in filter
+        design.
+        When False, generates a periodic window, for use in spectral analysis.
+
+    Returns
+    -------
+    out : array
+        The window, with the center value normalized to one (the value
+        one appears only if the number of samples is odd).
+
+    See Also
+    --------
+    kaiser, bartlett, blackman, hamming, hanning
+
+    References
+    ----------
+    .. [1] W. Carrara, R. Goodman, and R. Majewski, "Spotlight Synthetic
+           Aperture Radar: Signal Processing Algorithms" Pages 512-513,
+           July 1995.
+    .. [2] Armin Doerry, "Catalog of Window Taper Functions for
+           Sidelobe Control", 2017.
+           https://www.researchgate.net/profile/Armin_Doerry/publication/316281181_Catalog_of_Window_Taper_Functions_for_Sidelobe_Control/links/58f92cb2a6fdccb121c9d54d/Catalog-of-Window-Taper-Functions-for-Sidelobe-Control.pdf
+    """  # noqa: E501
```

Comment:
Copy the plotted example from chebwin?  Maybe show them on top of each other to show how one approximates the other?

Reply:
So I can copy down the example, but just to be sure, the images are inserted into the RST after sphinx autodoc runs, correct?

Reply:
> The main reason I say is because there is no import numpy as np so I think the example wouldn't run directly as is.

This is part of the example magic. Some imports like `import numpy as np` and `import matplotlib.pyplot as plt` IIRC are done automatically. So yes they should run, and you should be able to verify the output by looking at the CircleCI / Sphinx artfiacts link generated in the CI list below, [here](https://22018-1460385-gh.circle-artifacts.com/0/html-scipyorg/generated/scipy.signal.windows.taylor.html#scipy.signal.windows.taylor) is your current one and [here](https://22018-1460385-gh.circle-artifacts.com/0/html-scipyorg/generated/scipy.signal.windows.chebwin.html#scipy.signal.windows.chebwin) is the chebwin one)

Reply:
Ahh OK, great. Thanks for all the guidance on these issues!

One thing I noticed from the artifacts on the current build is that the first reference number is hyperlinked for some reason. Why is that one hyperlinked vs the second vs all the chebwin references? It seems to be linking to itself... the URL shows that it's the same page `#id1`...

Reply:
I think I see now, the references will only become hyperlinked if there is a callout to the reference somewhere in the documentation. Here reference [1] is called out above.